docs/beamrunner/beamrunner-2-install.md

```diff
@@ -0,0 +1,128 @@
+
+## Validating the Streams Runner installation
+
+The Streams Runner heavily relies on the `com.ibm.streams.beam` directory structure being preserved. To verify an installation, ensure that the translation and sdk JAR files appear as follows when you enter the following command:
+`ls $STREAMS_BEAM_TOOLKIT/lib`
+
+```
+> com.ibm.streams.beam.sdk.jar  com.ibm.streams.beam.translation.jar
+```
+
+If the JAR files are not present, set `STREAMS_BEAM_TOOLKIT` to `<streams-runner-extraction-location>/com.ibm.streams.beam-1.0.0/com.ibm.streams.beam`
+
+The Streams Runner directory tree structure
+```
+com.ibm.streams.beam-1.0.0/
+| - template.vcap
+| - com.ibm.streams.beam/
+| | - info.xml
+| | - toolkit.xml
+| | - com.ibm.streams.beam.transforms/
+| | - doc/
+| | - opt/
+| | - impl/
+| | | - lib/
+| | | | - com.ibm.streams.beam.runtime.jar
+| | - lib/
+| | | - com.ibm.streams.beam.sdk.jar
+| | | - com.ibm.streams.beam.translation.jar
+| - samples/
+| | - README.html
+| | - Makefile
+| | - README.md
+| | - src/
+| | - doc/
+| | - bin/
+| | | - streams-runner-env.sh
+| | - lib/
+| | | - com.ibm.streams.beam.samples.jar
+```
+
+## Creating a credentials file for your Streaming Analytics service
+
+To submit a Beam application to your Streaming Analytics service on Bluemix, you must create a JSON-formatted VCAP file that holds credentials and other information for the service. For a video demonstration of creating the credentials file, see [Creating a credentials file for the Streaming Analytics service on IBM Bluemix](https://ibm.box.com/s/qasw203e6gtdjpwu5ybmygcvstkr0xx8).
+
+To create a VCAP file for an existing Streaming Analytics service:
+
+1. Navigate to the folder where you installed the toolkit (`$STREAMS\_RUNNER\_HOME`) and copy the `template.vcap` file to a new file. Give the file a meaningful name and a file extension of `.vcap`.
+2. Copy the credentials of your Streaming Analytics service:
+  1. On the Streaming Analytics service page, click **Service credentials**.
+  2. If necessary, create a credential by clicking **New credential**. Use the default information and click **Add**.
+  3. Click **View credentials** for the credential that you want to use in your VCAP file. Click **Copy** to copy the credentials.
+3. Paste the copied credentials into the VCAP file that you created, replacing the following line:  
+    `<REMOVE THIS LINE AND INSERT CREDENTIALS HERE>`
+
+   Tip: You can have multiple credentials in the same file, as shown in the following example. When you run Beam applications, you must identify which credentials are used by placing the name value into the `--serviceName` parameter. For example, `--serviceName="alternate-service"` uses the second set of credentials shown in the example.
+

```

Comment:
If we have only one example, it should be of the most simple case possible, so it should have a single set of credentials and service name. 
Also, when would someone want to use multiple services? I guess adding something like "if you have x and you want to do Y, you might need multiple credentials in the same file,etc". Then we could show an additional example.